lib/fog/storage/google_json/real.rb

```diff
@@ -10,16 +10,12 @@ class Real
 
         def initialize(options = {})
           shared_initialize(options[:google_project], GOOGLE_STORAGE_JSON_API_VERSION, GOOGLE_STORAGE_JSON_BASE_URL)
+          @options = options.dup
           options[:google_api_scope_url] = GOOGLE_STORAGE_JSON_API_SCOPE_URLS.join(" ")
           @host = options[:host] || "storage.googleapis.com"
 
           # TODO(temikus): Do we even need this client?
```

Comment:
I was tempted to drop this and `@client` since they're not actually used.

Reply:
You use this API more than me so I am happy to drop this if there are no corner cases here 👍

Reply:
Let's leave this in for now.